Public/Src/Utilities/Instrumentation/LogGen/Generators/AriaV2.cs

```diff
@@ -94,13 +110,14 @@ private void WritePayload(LoggingSite site, LoggingSite.AddressedType item)
                     m_codeGenerator.Ln("foreach (var item in {0})", item.Address);
                     using (m_codeGenerator.Br)
                     {
+                        m_codeGenerator.Ln("var scrubbedPropertyName = {0}.AriaV2StaticState.ScrubEventProperty(item.Key);", GlobalInstrumentationNamespaceCommon);

```

Comment:
>ScrubEventProperty [](start = 93, length = 18)

Who knows if we have other fields that start with a number. What do you think about embedding the "don't start with a number" rule directly into ScrubEventProperty? That way we could keep the schema of our telemetry the same across platforms. Downside would be a one time breaking change.

Reply:
If you think this breaking change is no big deal, your suggestion would be preferred

Reply:
changed